web_chatter_position/static/src/js/web_chatter_position.esm.js

```diff
@@ -107,6 +107,10 @@ patch(FormCompiler.prototype, "web_chatter_position", {
                 if (!formSheetBgXml) {
                     return res
                 }
+                const formSheet = formSheetBgXml.querySelector(".o_form_sheet");
```

Comment:
@duyquyen96 xử giúp anh theo hướng này, để tránh ảnh hưởng đến chỗ khác
tìm phần từ chứa class `o_note_form_view` và xóa class `flex-nowrap h-100` của phần tử đó đi, thay bằng `flex-column`
nhớ note trong code là xử lý cho note
```suggestion
                const formSheet = formSheetBgXml.querySelector(".o_note_form_view");
```

Reply:
Dạ vâng ạ